docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "IncorrectEventABIInLibraries",
+        "summary": "Contract types used in events in libraries cause an incorrect hash in the ABI",
+        "description": "Instead of using the type 'address' for hashing, the actual contract name was used, leading to a wrong hash in the ABI.",
```

Comment:
```suggestion
        "description": "Instead of using the type `address` in the hashed signature, the actual contract name was used, leading to a wrong hash in the logs.",
```